src/sentry/static/sentry/app/plugins/components/pluginIcon.tsx

```diff
@@ -88,10 +93,15 @@ const IntegrationIcon = styled('div')`
   background-image: url(${p => p.image});
 `;
 
-class PluginIcon extends React.Component {
+type Props = {
```

Comment:
These props are missing all of the `HTMLDivElement` attributes, since we spread props into the styled component.

Reply:
Do we need to spread props into all styled components? I know it could happen but is it something we need here?